### PR TITLE
Use the new file-level option of solidity-protobuf

### PR DESCRIPTION
**Pull request by siburu**

### CODE REVIEW

**Comment by bluele** [comment]
ref. https://github.com/datachainlab/solidity-protobuf/pull/33
(This comment intent to just link the corresponding PR)

File: scripts/solpb.sh [hunk 1]
```diff
@@ -6,10 +6,14 @@ if [ -z "$SOLPB_DIR" ]; then
     exit 1
 fi
 
+GEN_DIR="$(pwd)/generated"
+mkdir $(pwd)/generated
```
Comment (bluele) [inline scripts/solpb.sh:10]:
```suggestion
mkdir "$GEN_DIR"
```

Comment (bluele) [reply scripts/solpb.sh:10]: @siburu I've made a small suggestion. (However, if you try to apply it using the github's feature, I guess the commit may not contain Signed-off.)

Comment (siburu) [reply scripts/solpb.sh]: @bluele Thank you! I've just fixed it.

**Review by bluele — APPROVED** [review]
:+1:

**Pull request merged**